src/platforms/dotnet/guides/aspnet/performance/troubleshooting.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Troubleshooting
+sidebar_order: 100
+description: "Learn more about how to troubleshoot performance issues with the ASP.NET SDK."
+---
+
+## Transactions get grouped together as `GET /*/*.`
+
+The SDK creates the transaction name as follows
+
+```csharp
+var method = httpContext.Request.HttpMethod;
+var path = httpContext.Request.Path;
+
+var transactionName = $"{method} {path}";
+
+// Since the name is derived from the path and potentially contains identifiers it is marked as such
+transactionContext.NameSource = TransactionNameSource.Url;
+```
+
+Since the `URL` might contain potential identifiers these transactions have their name stripped.
```

Comment:
```suggestion
Since the `URL` might contain potential identifiers, such as `GET /users/123`, the backend tries to remove values with high variations. Leading it to `GET /users/*`. Sometimes, when there are many paths, it can result in `GET /*/*`.
```